kinds/dashboard/dashboard_kind.cue

```diff
@@ -16,37 +16,47 @@ lineage: schemas: [{
 	schema: {
 		spec: {
 			// Unique numeric identifier for the dashboard.
-			// TODO must isolate or remove identifiers local to a Grafana instance...?
 			id?: int64
+
 			// Unique dashboard identifier that can be generated by anyone. string (8-40)
 			uid?: string
+
 			// Title of dashboard.
 			title?: string
+
 			// Description of dashboard.
 			description?: string
+
 			// This property should only be used in dashboards defined by plugins.  It is a quick check
-			// to see if the version has changed since the last time.  Unclear why using the version property
-			// is insufficient.
-			revision?: int64 @grafanamaturity(NeedsExpertReview)
+			// to see if the version has changed since the last time.
+			revision?: int64
+
 			// ID of a dashboard imported from the https://grafana.com/grafana/dashboards/ portal
-			gnetId?: string @grafanamaturity(NeedsExpertReview)
+			gnetId?: string
+
 			// Tags associated with dashboard.
-			tags?: [...string] @grafanamaturity(NeedsExpertReview)
+			tags?: [...string]
+
 			// Theme of dashboard.
-			style: "light" | *"dark" @grafanamaturity(NeedsExpertReview)
+			// Accepted values are "light" (light theme), "dark" (dark theme, default).
```

Comment:
```suggestion
			// Default value: dark.
```
Because the possible values are automatically added (see the `.md` file above)

Reply:
I like it!